cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -55,10 +55,10 @@ func atoiUint64(t *testing.T, s string) uint64 {
 type extractMetricsMatch *regexp.Regexp
 
 // extractMetricsLogsMatch is a regular expression to extract metrics from log messages.
-var extractMetricsLogsMatch = extractMetricsMatch(regexp.MustCompile(`score=(\d+), coverage=(\d+), files-executed=(\d+), generate-tests-for-file-character-count=(\d+), processing-time=(\d+), response-character-count=(\d+), response-no-error=(\d+), response-no-excess=(\d+), response-with-code=(\d+)`))
+var extractMetricsLogsMatch = extractMetricsMatch(regexp.MustCompile(`cost=\d+(?:\.\d+)?, score=(\d+), coverage=(\d+), files-executed=(\d+), generate-tests-for-file-character-count=(\d+), processing-time=(\d+), response-character-count=(\d+), response-no-error=(\d+), response-no-excess=(\d+), response-with-code=(\d+)`))
 
 // extractMetricsCSVMatch is a regular expression to extract metrics from CSV rows.
-var extractMetricsCSVMatch = extractMetricsMatch(regexp.MustCompile(`(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+)`))
+var extractMetricsCSVMatch = extractMetricsMatch(regexp.MustCompile(`\d+(?:\.\d+)?,(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+),(\d+)`))
```

Comment:
Please add comments that we don't match the cost as a group since it is just a property the model that we carry along for informational purposes.